lib/moab/storage_object_validator.rb

```diff
@@ -63,6 +65,7 @@ def version_directories
 
     def check_correctly_named_version_dirs
       errors = []
+      errors << result_hash(MISSING_DIR, 'no versions exist') unless version_directories.count > 0
```

Comment:
👍

Reply:
Oh nice - I added a check for MISSING_DIR in my PR too - i guess we can just resolve them when we merge. Your solution is more elegant than mine! :D

Reply:
yeah - I like this solution better.